packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -488,10 +494,13 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
                     }
                 }
             } as ReplaceOperation));
+            // Replace the text.
             await widget.editor.replaceText({
                 source,
                 replaceOperations
             });
+            // Save the text replacement changes in the editor.
+            await widget.saveable.save();
```

Comment:
You should use `shell.save` to go through proper shell `save` cycle.

Reply:
The save() and saveAll() methods in `shell` only save the tracked dirty widget. However, since the widgets are not opened, they are not tracked. I am wondering if `shell.save()` can be used in this case?

Reply:
> You should use `shell.save` to go through proper shell `save` cycle.

The save() and saveAll() methods in shell only save the tracked dirty widget. However, since the widgets are not opened, they are not tracked. I am wondering if shell.save() can be used in this case?

Reply:
ok